Gems/Atom/RHI/DX12/Code/Source/RHI/DescriptorContext.cpp

```diff
@@ -197,6 +178,14 @@ namespace AZ
             if (shaderResourceView.IsNull())
             {
                 shaderResourceView = AllocateHandle(D3D12_DESCRIPTOR_HEAP_TYPE_CBV_SRV_UAV, D3D12_DESCRIPTOR_HEAP_FLAG_NONE, 1);
+                if (shaderResourceView.IsNull())
+                {
+                    AZ_Assert(
+                        false,
+                        "Descriptor heap ran out of memory for descriptor handles. Please consider increasing the first value of "
+                        "DESCRIPTOR_HEAP_TYPE_CBV_SRV_UAV within platformlimits.azasset file for dx12.");
```

Comment:
Should `DESCRIPTOR_HEAP_TYPE_CBV_SRV_UAV` be `D3D12_DESCRIPTOR_HEAP_TYPE_CBV_SRV_UAV` instead? (and for the rest of the allocation asserts below)